src/Hilke.KineticConvolution/DoubleAlgebraicNumber/InvalidityManagementMode.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Hilke.KineticConvolution.DoubleAlgebraicNumber
+{
+    public enum InvalidityManagementMode
+    {
+        Silent = 0,
+        ThrowException = 1
+    }
```

Comment:
```suggestion
    public enum InvalidConversionPolicy
    {
        Throw = 0,
        IgnoreSilently = 1
    }
```
0 is the default value of an `enum`, and I prefer that by default we are more vocal upon invalid conversion.